package.json

```
@@ -12,13 +12,14 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "turbo run build",
+    "build": "cd packages/sdk && yarn typechain && cd ../.. && turbo run build lint",
     "build:app": "cd packages/sdk && yarn typechain && turbo run build --filter=@stackly/sdk --filter=app",
     "build:landing": "cd packages/sdk && yarn typechain && turbo run build --filter=@stackly/sdk --filter=landing",
```

```suggestion
    "build": "cd packages/sdk && bun typechain && cd ../.. && turbo run build lint",
    "build:app": "cd packages/sdk && bun typechain && turbo run build --filter=@stackly/sdk --filter=app",
    "build:landing": "cd packages/sdk && bun typechain && turbo run build --filter=@stackly/sdk --filter=landing",
```

Let's change it to bun!